scripts/kfctl.sh

```diff
@@ -148,6 +184,9 @@ if [ "${COMMAND}" == "init" ]; then
 fi
 
 source ${ENV_FILE}
+source "${DIR}/util.sh"
```

Comment:
Didn't we already source the files?

Reply:
This was restructured, hence the need to move the source statements.